Dockerfile

```diff
@@ -5,7 +5,10 @@ COPY package*.json ./
 RUN apk add --no-cache python3 make g++
 
 RUN npm ci --ignore-scripts
+
+ENV TARGET_ARCH="arm64"
```

Comment:
should it be this or 386?  386 worked for me.  really we need to check the effective architecture and map it